Makefile

```diff
@@ -1,5 +1,7 @@
 .PHONY: all clean install check test lint-install lint
 
+INCOMPATIBLE_ESLINT_VERSIONS=$(shell node --version | egrep 'v0.[2-9]' | cut -d '.' -f 1-2)
```

Comment:
This misses `v0.1`, since it's slightly incovenient to disallow 1 but allow v0.10 and v0.12. I hope that's alright and no one's still running v0.1.